src/main/java/io/mosip/mimoto/constant/SwaggerLiteralConstants.java

```diff
@@ -0,0 +1,77 @@
+    public static final String IDP_GET_TOKEN_DESCRIPTION = "This endpoint allow you to retrieve the access token in exchange for authorization Code" ;
+
+    /*  Issuers Controller */
+    public static final String ISSUERS_NAME = "Issuers" ;
+    public static final String ISSUERS_DESCRIPTION = "All the Issuers Related Endpoints" ;
+    public static final String ISSUERS_GET_ISSUERS_SUMMARY = "Retrieve All Onboarded Issuers" ;
+    public static final String ISSUERS_GET_ISSUERS_DESCRIPTION = "This endpoint allow you to retrieve all the onboarded issuers" ;
+    public static final String ISSUERS_GET_SPECIFIC_ISSUER_SUMMARY = "Retrieve Specific Issuer's Config" ;
+    public static final String ISSUERS_GET_SPECIFIC_ISSUER_DESCRIPTION = "This endpoint allow you to retrieve the Complete configuration of the specific issuer" ;
+    public static final String ISSUERS_GET_ISSUER_WELLKNOWN_SUMMARY = "Retrieve Specific Issuer's Well known" ;
+    public static final String ISSUERS_GET_ISSUER_WELLKNOWN_DESCRIPTION = "This endpoint allow you to retrieve the well known of the specific issuer" ;
+
+    /*  Prensentation Controller */
+    public static final String PRESENTATION_NAME = "Presentation" ;
+    public static final String PRESENTATION_DESCRIPTION = "All the Online Sharing Related Endpoints" ;
+    public static final String PRESENTATION_AUTHORIZE_SUMMARY = "Perform the Authorization" ;
+    public static final String PRESENTATION_AUTHORIZE_DESCRIPTION = "This endpoint allow you to redirect the token back to the caller post authorization" ;
+
+    /*  Resident Service Controller */
+    public static final String RESIDENT_NAME = "Resident Service" ;
+    public static final String RESIDENT_DESCRIPTION = "All the Resident Service Related Endpoints" ;
+    public static final String RESIDENT_REQUEST_OTP_SUMMARY = "Request for OTP" ;
+    public static final String RESIDENT_REQUEST_OTP_DESCRIPTION = "This endpoint allow you to request OTP for credential Download" ;
+    public static final String RESIDENT_REQUEST_INDIVIDUALID_OTP_SUMMARY = "Request OTP for retrieving individual Id" ;
+    public static final String RESIDENT_REQUEST_INDIVIDUALID_OTP_DESCRIPTION = "This endpoint allow you to request OTP to download individual ID" ;
+    public static final String RESIDENT_GET_INDIVIDUALID_SUMMARY = "Retrieve Individual Id using AID" ;
+    public static final String RESIDENT_GET_INDIVIDUALID_DESCRIPTION = "This endpoint allow you to retrieve the Individual Id using AID" ;
+
+
+    /*  Verifiers Controller */
+    public static final String VERIFIERS_NAME = "Verifiers" ;
+    public static final String VERIFIERS_DESCRIPTION = "All the Verifiers Related Endpoints" ;
+    public static final String VERIFIERS_GET_VERIFIERS_SUMMARY = "Retrieve All Trusted Verifiers" ;
+    public static final String VERIFIERS_GET_VERIFIERS_DESCRIPTION = "This endpoint allow you to retrieve all the Trusted Verifiers" ;
```

Comment:
T and V in small case for Trusted Verifiers

Reply:
Updated